common/djangoapps/util/testing.py

```diff
@@ -1,9 +1,18 @@
 import sys
 
+from datetime import datetime
+from pytz import UTC
 from mock import patch
 
 from django.conf import settings
 from django.core.urlresolvers import clear_url_caches, resolve
+from openedx.core.djangoapps.course_groups.models import CourseUserGroupPartitionGroup
```

Comment:
Well, they're only ever used by the LMS right now, but in the future, it might be useful to have them around as setup mixins for particular course structures in CMS as well. I can poke around for a good home for them in `openedx` if that would be preferable.